pkg/user/tx_client_test.go

```diff
@@ -47,8 +48,6 @@ func (suite *TxClientTestSuite) SetupSuite() {
 	suite.Require().NoError(err)
 	suite.txClient, err = user.SetupTxClient(suite.ctx.GoContext(), suite.ctx.Keyring, suite.ctx.GRPCClient, suite.encCfg, user.WithGasMultiplier(1.2))
 	suite.Require().NoError(err)
-	// FIXME: Temporary way of querying the raw log.
-	// TxStatus will natively support this in the future.
 	suite.serviceClient = sdktx.NewServiceClient(suite.ctx.GRPCClient)
```

Comment:
Do we still need the service client here?

Reply:
yes because we are polling GasWanted GasUsed